component/core/src/test/java/org/exoplatform/social/core/jpa/storage/dao/IdentityDAOTest.java

```diff
@@ -83,7 +83,7 @@ public void testGetAllIds() {
 
     // Then
     assertNotNull(allIds);
-    assertEquals(3 + initialSize, allIds.size());
+    assertEquals(0 + initialSize, allIds.size());
```

Comment:
why this change, can you explain please what happens without this change ?

Reply:
Reverted